src/Symfony/Component/Mime/Tests/Header/UnstructuredHeaderTest.php

```diff
@@ -244,4 +244,20 @@ public function testGetBody()
         $header = new UnstructuredHeader('Subject', 'test');
         $this->assertEquals('test', $header->getBody());
     }
+
+    public function testInReplyToIsUnstructuredHeader()
+    {
+        $email = new Email();
+        $headers = $email->getHeaders();
+        $headers->addTextHeader('In-Reply-To', 'foobar');
```

Comment:
use `addHeader`, otherwise your test won't check the fix (`addTextHeader` always adds an unstructured header, and so your tests also pass before your patch)

Reply:
OK. But without the code change it did fail:
> Symfony\Component\Mime\Exception\LogicException: The "In-Reply-To" header must be an instance of "Symfony\Component\Mime\Header\IdentificationHeader" (got "Symfony\Component\Mime\Header\UnstructuredHeader").